app/src/main/java/com/example/wearit/ui/SettingsScreen.kt

```diff
@@ -0,0 +1,126 @@
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.example.wearit.components.ButtonType
+import com.example.wearit.components.MasterButton
+
+@Composable
+fun SettingsScreen(
+    isAppInDarkTheme: Boolean,
+    switchTheme: (darkTheme: Boolean) -> Unit,
+    goToPicker: () -> Unit
+) {
+    Scaffold(
+        content = {
+            SettingsContent(
+                isAppInDarkTheme = isAppInDarkTheme,
+                switchTheme = switchTheme
+            )
+        },
+        bottomBar = {
+            SettingsBottomBar(
+                goToPicker = goToPicker
+            )
+        }
+    )
+}
+
+@Composable
+fun SettingsContent(
```

Comment:
dodałbym tutaj dividery takie jak są w projekcie na figmie (pod toggle dark theme i  zaraz nad przyciskiem "Back"). No i może popróbować coś z tłem bo to białe tak sobie wygląda, ale już tekst jest czerwony więc może być ciężko, dlatego jeśli nic się nie uda wykombinować to trudno

Reply:
Dodałem dividery, a tła nie mogę ustawić, bo z jakiegoś powodu tło na komponencie Scaffold działa inaczej niż na innych komponentach i nie wiem jak ustawić tam gradient (stały kolor potrafię, ale to nie wygląda najlepiej)